src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java

```diff
@@ -30,6 +30,7 @@
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.ChplDownloadPage;
```

Comment:
Is this import needed here? I don't think any of the definitions from download page are being used in upload-confirm workflow

Reply:
Updated.